.NET/Microsoft.Recognizers.Text.Number/Parsers/BaseCJKNumberParser.cs

```diff
@@ -477,7 +477,15 @@ private double GetIntValue(string intStr)
             if (Config.NegativeNumberSignRegex.IsMatch(intStr))
             {
                 isNegative = true;
-                intStr = intStr.Substring(1);
+                //I'm so sorry thie code because i want to use 'NegativeNumberTermsRegex' but it is not in the CJKInterface
```

Comment:
You can add it to a config class like EnglishNumberParserConfiguration.cs and here you can get from the config class.
All expressions need to be defined in the source YAML files for that language.

Reply:
This if/else could also be its own method in BaseCJKNumberParser.cs for ease of reading.